media-gfx/openscad/openscad-2015.03_p2-r4.ebuild

```diff
@@ -1,7 +1,7 @@
-# Copyright 1999-2017 Gentoo Foundation
+# Copyright 1999-2018 Gentoo Authors
 # Distributed under the terms of the GNU General Public License v2
 
-EAPI=6
+EAPI=7
 
 inherit elisp-common qmake-utils xdg-utils
```

Comment:
if you switch to `xdg`, you can drop both `pkg_postinst` and `pkg_postrm`

Reply:
As of today, the openscad-2015.03_p2 ebuild has already been dropped from the main tree.
Switched to xdg and removed pkg_post{inst,rm}